### DISCUSSION TITLE
How to extract the peak coverage score?

### DISCUSSION THREAD

**Discussion by woshiyangsi** [GreenleafLab/ArchR#1428] [General]
Hi @rcorces ,
       I want to extract the peak coverage score to plot heatmap. Do you know how to extract the peak coverage score?

Looking forward to your reply.
Siyu

**Comment by rcorces** [comment]
Sorry for the delay in my reply. I think this depends on what you mean by peak coverage score and whether you want this metric on a per-cell basis (i.e. single-cell resolution) or on a per-group basis (i.e. pseudo-bulk resolution).

If you are trying to get this per-cell, then you just want the values stored in `PeakMatrix`.
If you are trying to get this per-group then you want to use `getGroupSE()`.
